monocle/bounds.py

```diff
@@ -38,12 +38,21 @@ def __init__(self, polygon=conf.BOUNDARIES):
         self.south, self.west, self.north, self.east = polygon.bounds
         self.center = polygon.centroid.coords[0]
         self.multi = False
+        self.prepared = False
+        try:
+            from shapely.prepared import prep
+            self.prepared_boundaries = prep(polygon)
+            self.prepared = True
+        except:
```

Comment:
Is it possible for Shapely to not have GEOS functionality? If not, we can get rid of this `try`/`except` and move the import down near the other shapely import. Otherwise change this line to:
```python
except Exception:
```
or whichever exception occurs- we don't want to catch non-`Exception`s like `KeyboardInterrupt`.